lib/fs.js

```diff
@@ -6,6 +6,7 @@
 const constants = process.binding('constants').fs;
 const util = require('util');
 const pathModule = require('path');
+const internalFs = require('internal/fs');
```

Comment:
It would, until the next few things are extracted and we get a refactor doing this same thing. Do we have precedence on this with internal? I'm not a fan.